tasks/ceph_manager.py

```diff
@@ -793,7 +793,11 @@ def tmp(x):
         pools = self.list_pools()
         self.pools = {}
         for pool in pools:
-            self.pools[pool] = self.get_pool_property(pool, 'pg_num')
+            # we may race with a pool deletion; ignore failures here
+            try:
+                self.pools[pool] = self.get_pool_property(pool, 'pg_num')
+            except:
```

Comment:
This should be as specific as possible; bare ``except:`` clauses should never be necessary. Here's why the're bad:
https://docs.python.org/2/howto/doanddont.html#except